### PR TITLE
Dockerize boodle

### PR DESCRIPTION
**Pull request by LukasWerfel**
Hi @manuel-uberti,

I wanted to start playing around with boodle a bit. I am trying to get my docker setup running. 
This installs all dependencies and starts everything up. 

I only have two problems:
1. Can I somehow set the [db host](https://github.com/manuel-uberti/boodle/blob/master/conf/config.edn#L1) by environment variable? Because in docker it would live in another host (called `db`).

2. Is there a way to automaticly populate the DB with the [sql script](https://github.com/manuel-uberti/boodle/blob/master/resources/sql/model.sql) if the entries are not existent yet? Or do I have to do this manually?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,12 @@ unit](https://www.freedesktop.org/software/systemd/man/systemd.unit.html) that
 executes `etc/deploy.sh` to update, build and run boodle automatically on my
 home server.
 
+### Docker
+You can also run boogle with docker. In this case you only need to run
```
Comment (manuel-uberti) [inline README.md]:
```suggestion
You can also run boodle with docker. In this case you only need to run
```

**Comment by LukasWerfel** [comment]
@manuel-uberti would you be generally interested in this, though? Or you do not think it be worth it?

**Comment by manuel-uberti** [comment]
To be honest, I do not plan to use docker with boodle. It's been deployed on my home server for more than a year now and I'm happy with this setup.

If adding a docker option is not too much of an effort for you, do it. I'll be happy to see boodle growing in interesting directions.

On the other hand, if you prefer to work on your own fork of boodle, it's fine with me. I open-sourced this project because I wanted to give something back to the Clojure community, so if it's useful to someone I can only be happy.

**Comment by manuel-uberti** [comment]
Hi @LukasWerfel, are you still interested in working on this? Or do you prefer to consider this PR closed?

**Comment by LukasWerfel** [comment]
I think it's fine then, thanks for your time!

**Pull request closed without merging**